vizro-core/examples/default/app_dev.py

```diff
@@ -41,8 +40,12 @@ def create_benchmark_analysis():
                 title="Click on a cell in country column:",
                 figure=dash_ag_grid(
                     data_frame=df,
+                    className="ag-theme-alpine vizro",
```

Comment:
The className I would move to the existing dash_ag_grid and define as a default there if the user hasn't specified anything

Reply:
Yes, this is alrady implemented as part of the docs PR